src/renderer/components/multimachinebackups/SelectSiteBackup.tsx

```diff
@@ -10,7 +10,7 @@ import { actions, store, useStoreSelector } from '../../store/store';
 import { ErrorBannerContainer } from './ErrorBannerContainer';
 import styles from './SelectSiteBackup.scss';
 
-interface Props {
+interface Props extends LocalRenderer.RouteComponentProps {
```

Comment:
This is exported now in Local Core, will need to link up API to test with `nps types.linkAPI` until a release republishes that though